lib/internal/cluster/round_robin_handle.js

```diff
@@ -16,9 +17,14 @@ function RoundRobinHandle(key, address, port, addressType, fd) {
 
   if (fd >= 0)
     this.server.listen({ fd });
-  else if (port >= 0)
-    this.server.listen(port, address);
-  else
+  else if (port >= 0) {
+    this.server.listen({
+      port,
+      host: address,
+      // Currently, net module only supports `ipv6Only` option in `flags`.
+      ipv6Only: Boolean(flags | constants.UV_TCP_IPV6ONLY),
```

Comment:
This looks wrong. `Boolean(flags | constants.UV_TCP_IPV6ONLY)` is always true. Did you mean to use `&`?

Reply:
Yes, a terrible mistake. I'm going to update it and add a test of falsy `ipv6Only` scenario.